.github/workflows/container.yml

```diff
@@ -28,8 +28,8 @@ jobs:
           node-version-file: '.nvmrc'
           cache: 'yarn'
 
-      - name: Generate production build
-        run: make build
+      - name: Generate production bundle
+        run: make riffraff-bundle
```

Comment:
Can we now delete
- `riffraff-publish` task from the makefile
- `deploy` task from the makefile
- `publish-assets.sh`

Could be a follow up